python/pyarrow/filesystem.py

```diff
@@ -100,44 +113,72 @@ def stat(self, path):
 
     def rm(self, path, recursive=False):
         """
-        Alias for FileSystem.delete
+        Alias for FileSystem.delete.
         """
         return self.delete(path, recursive=recursive)
 
     def mv(self, path, new_path):
         """
-        Alias for FileSystem.rename
+        Alias for FileSystem.rename.
         """
         return self.rename(path, new_path)
 
     def rename(self, path, new_path):
         """
-        Rename file, like UNIX mv command
+        Rename file, like UNIX mv command.
 
         Parameters
         ----------
-        path : string
-            Path to alter
-        new_path : string
-            Path to move to
+        path : str
+            Path to alter.
+        new_path : str
+            Path to move to.
         """
         raise NotImplementedError('FileSystem.rename')
 
     def mkdir(self, path, create_parents=True):
+        """
+        Creates a directory.
```

Comment:
```suggestion
        Create a directory.
```

Please use infinitives ("create"), not 3rd person present ("creates").